XamlControlsGallery/ControlPages/ListViewPage.xaml

```diff
@@ -262,6 +262,31 @@ source code for the XAML Controls Gallery. -->
             </local:ControlExample.Substitutions>
         </local:ControlExample>
 
+        <local:ControlExample HeaderText="ListView with Filtering"
+                              XamlSource="ListView/ListViewSample4_xaml.txt"
+                              CSharpSource="ListView/ListViewSample4_cs.txt">
+            <ListView 
+                x:Name="FilteredListView"
+                ItemTemplate="{StaticResource ContactListViewTemplate}"
+                SelectionMode="Single"
+                ShowsScrollingPlaceholders="True"
+                Height="400"
+                Width="400"
+                BorderThickness="1"
+                BorderBrush="{ThemeResource SystemControlForegroundBaseMediumLowBrush}">
+            </ListView>
+
+            <local:ControlExample.Options>
+                <StackPanel Width="200">
+                    <TextBlock Margin="8,8,8,4" Style="{ThemeResource BaseTextBlockStyle}">Filter by...</TextBlock>
+                    <TextBox x:Name="FilterByFirstName" Width="150" Header="First name" Margin="8" HorizontalAlignment="Left" TextChanged="OnFilterChanged"></TextBox>
```

Comment:
Also, wrapping these lines makes them easier to read in a code review without having to scroll.